schema.ts

```diff
@@ -1245,6 +1246,18 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * @hidden
    */
   groupIdProperty?: PropertyIdentifier<K>;
+
+  /**
+   * The name of the property within {@link ObjectSchemaDefinition.properties} that can be be interpreted as
+   * a large blob of unstructured content.
+   *
+   * This should consist of text or markdown content.
+   *
+   * Must be a {@link ValueType.String} property.
+   * TODO(sam): Unhide this
+   * @hidden
+   */
+  unstructuredContentProperty?: PropertyIdentifier<K>;
```

Comment:
I'd consider a different name. `unstructured` doesn't really say much, and feels like a very inside-baseball term. Some alternatives:
- `textContextProperty`
- `bodyTextProperty`
- `contentProperty`
- `bodyProperty`

Reply:
bodyProperty to me is very clear

Reply:
I vote `bodyTextProperty` (and `bodyProperty` is a close 2nd).